### ISSUE TITLE
Auto-setup url authentication is same for all App provider companies

### ISSUE THREAD

**Issue by dhiren-singh-007** [eclipse-tractusx/portal-backend#1151]
### Current Behavior
Currently App provider can set the Auto-Setup Url only and when backend calls this endpoint ,it sends the token generated by client which is coming from environment variable.
![image](https://github.com/user-attachments/assets/c2a5c42b-0040-4acd-9b86-470db4a87011)
So that mean App provider needs to set Central idp from operator as authority on their auto setup url.
And all App providers will have to share same client id and secret for this.

### Expected Behavior
App provider should be able to set the Auth url , client Id and secret so that portal BE can access the auto setup url by using these details same like OSP callback URL
![image](https://github.com/user-attachments/assets/ad36ada6-1cad-4a10-815e-c3566d2406cf)

@MaximilianHauer @evegufy @Phil91 @ntruchsess 
As we discussed in open meeting , i have created this issue.
Please let me know if i am missing anything here.

**Comment by Phil91** [comment]
@dhiren-singh-007 I agree, we should give the providers the possibility to setup their own user / authentication url.

@MaximilianHauer fyi

I've also just added the following issue: https://github.com/eclipse-tractusx/portal-backend/issues/1175

**Comment by evegufy** [comment]
@dhiren-singh-007 could you please also update the documentation for auto-setup in portal-assets?

**Comment by evegufy** [comment]
even tough I agree with the proposed change, as the current behaviour is concept-wise works-as-designed, this should be considered a `fix` but a. `feat`

**Comment by evegufy** [comment]
> [@dhiren-singh-007](https://github.com/dhiren-singh-007) could you please also update the documentation for auto-setup in portal-assets?

ignore, didn't notice the already available PR

**Comment by evegufy** [comment]
@dhiren-singh-007 sig-release issue needed for proper testing
